feat: New contributing guidelines and usage outline (#127)
thebrowsercompany/swift-webdriver: CONTRIBUTING.md

> ## Contribution Guidelines

Reviewer: ```suggestion
## Contribution Guidelines

```

Welcome to swift-webdriver a Swift library for UI automation of Windows applications through the [WebDriver](https://w3c.github.io/webdriver/) protocol, similar to [Selenium](https://www.selenium.dev/), [Appium](http://appium.io/) or the [Windows Application Driver](https://github.com/microsoft/WinAppDriver). We welcome contributions, however before contributing, please read this entire document to ensure quality pull requests.

> ##Testing

Reviewer: ```suggestion
## Testing

```

All UI tests are located in the `Tests` directory. Build and run tests using `swift build` and `swift test`, or use the [Swift extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=sswg.swift-lang).

For additional examples, refer to the `Tests\WebDriverTests` directory.

> ##Bug Submissions

Reviewer: ```suggestion
## Bug Submissions

```

If you've found a bug in the project feel free to open an issue [here](https://github.com/thebrowsercompany/swift-webdriver/issues/new). Please follow the issues template provided.

> ##Enhancement Requests

Reviewer: ```suggestion
## Enhancement Requests

```

To submit an enhancement open a new issue with the `enhancements` tag with details for what you think would be good to add.

> ##Style Guide

Reviewer: ```suggestion
## Style Guide

```

## Contribution Guidelines

> Welcome to swift-webdriver a Swift library for UI automation of apps and browsers via communication with [WebDriver](https://w3c.github.io/webdriver/) endpoints, such as [Selenium](https://www.selenium.dev/), [Appium](http://appium.io/) or the [Windows Application Driver](https://github.com/microsoft/WinAppDriver). We welcome contributions of all kinds however before contributing please read this entire document to ensure quality pull requests

Reviewer: ```suggestion
Welcome to swift-webdriver a Swift library for UI automation of Windows applications through the [WebDriver](https://w3c.github.io/webdriver/) protocol, similar to [Selenium](https://www.selenium.dev/), [Appium](http://appium.io/) or the [Windows Application Driver](https://github.com/microsoft/WinAppDriver). We welcome contributions, however before contributing, please read this entire document to ensure quality pull requests.
```
